app/webpage/homepage.html

```diff
@@ -0,0 +1,312 @@
+    <script src="main.js"></script>
+  </head>
+  <body>
+<!-- lgin -->
+    <div class="top container-fluid" id="login">
+    
+<!-- lgin-user -->
+      <div class="row row-eq-height signin" id="username">
+        <div class="col-md-2 col-md-offset-5 front">  
+          <h3>Email:</h3>
+          <div class="level-3 input">
+            <input type="text" id="email" value="" size=35>
+          </div>
+        </div>
+        <div class="col-md-1 front">
+          <button class="user-verify" id="log-email">Next</button> 
+        </div>
+      </div>
+      
+<!-- lgin-pswd -->
+      <div class="row row-eq-height signin" id="password">
+        <div class="col-md-2 col-md-offset-5 front">  
+          <h3>Password:</h3>
+          <div class="level-3 input">
+            <input type="password" id="pswd" value="" size=35>
+          </div>
+        </div>
+        <div class="col-md-1 front">
+          <button class="user-verify" id="log-pswd">Login</button> 
+        </div>
+      </div>
+      
+<!-- lgin-npwd -->
+      <div class="row row-eq-height signin" id="sign-up">
+        <div class="col-md-3 col-md-offset-5 front">
+          <h3>Create Password:</h3>
+          <div class="row row-eq-height create">
+            <div class="col-md-9">
+              <h4>Enter Password:</h4>
+              <div class="level-3 input">
+                <input type="password" id="pswd-create" value="" size=35>
+              </div>
+              <h4>Confirm Password:</h4>
+              <div class="level-3 input">
+                <input type="password" id="confirm" value="" size=35>
+              </div>
+            </div>
+            <div class="col-md-3">
+              <button class="user-verify" id="log-create">Sign Up</button> 
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+    
+<!-- plan -->
+    <div class="container-fluid" id="week-scheduler">
+    
+<!-- plan-head -->
+      <div class="row row-eq-height" id="title-bar">
+        <div class="col-md-8 col-md-offset-2">
+<!-- plan-head-01 -->
+          <h1 id="header">[user]'s Tasklist</h1>
+        </div>
+<!-- plan-head-02 -->
+        <div class="col-md-2">
+          <div class="dropdown" id="option-div">
+            <button id="option-btn" class="btn btn-primary dropdown-toggle" type="button" data-toggle="dropdown"></button>
+            <ul class="dropdown-menu">
+              <li id="pswd-change"><p>Change your password</p></li>
+              <li id="end-user"><p>Delete your account</p></li>
+              <li class="divider"></li>
+              <li id="help"><p>Help</p></li>
+            </ul>
+          </div>
+        </div>
+      </div>
+     
+<!-- plan-wksl, plan-tlbr -->
+      <div class="row row-eq-height" id="body">
+<!-- plan-wksl -->
+        <div class="col-md-9" id="schedule">
+        
+          <div class="week weekend Sunday">
+            <div class="day">Sunday</div>
+            <div class="task">
```

Comment:
Of sorts. I wanted to get the general layout on the webpage, and this was the best shorthand I knew to do, I was aiming at including jQuery there when I addressed pulling tasks from the database.. So yes. I'll be going in and changing things after I get the capability to pull the user's saved tasks, and I was thinking that jQuery would certainly be less bulky, but I'd have to come up with a way to address scalability.
But I'd be happy to hear better ideas, especially than what's there now. This serves a much better purpose looking at it on the webpage than it does the html code, I'd be more than happy to change it.